data/campaigns/Delfadors_Memoirs/scenarios/23_Epilogue.cfg

```diff
@@ -8,13 +8,13 @@
     [story]
         [part]
             music=battle-epic.ogg
-            story=_ "Delfador returned to Weldyn, but not before secreting the Book of Crelanu in a place known only to him. To his surprise, Asheviere made no further mention of the events in the manor. But not long after, orcish raids increased to a degree that threatened the Kingdom. The King mustered his armies to deal with the orcish menace, leading to the great battle at the Ford of Abez that was lost through betrayal."
+            story=_ "Delfador returned to Weldyn, but not before secreting the Book of Crelanu to a place known only to him. To his surprise, Asheviere made no further mention of the events in the manor. But not long after, orcish raids increased to a degree that threatened the Kingdom. The King mustered his armies to deal with the orcish menace, leading to the great battle at the Ford of Abez that was lost through betrayal."
```

Comment:
> To his surprise, Asheviere made no further mention of the events in the manor.

Another thing I just noticed (that was already present in the original text before this PR). Would it make sense to drop the 'further' in this sentence? As it stands, it implies to me that Asheviere had made some mention of the events in the manor (Delfador's attack on Sagus) but is no longer discussing it. Without the 'further' I interpret this as no mention at all, which seems to make more sense. It's a subtle difference, I know, but I thought it worth bringing up if this is supposed to be a big revision of the prose (forthcoming rewrite of the campaign notwithstanding).

Reply:
Since I had no feedback on this suggestion (in contrast with the other one) I'm assuming there's no objections and updated this in 0c13523544d36138dd58a780d2dd4197ba5bef8b (I didn't think it was big enough to warrant a full PR).